### ISSUE TITLE
Missing Visual Focus on Global Alerts

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4338]
Global Alerts should be Keyboard Accessible

![Screen Shot 2019-08-29 at 9 15 24 PM](https://user-images.githubusercontent.com/15113219/63986688-10b09a00-caa3-11e9-98ac-d71480011599.png)

![Screen Shot 2019-08-29 at 9 16 03 PM](https://user-images.githubusercontent.com/15113219/63986697-14dcb780-caa3-11e9-8b44-3b73135a4bf6.png)

**Comment by KateChuen** [comment]
Tested on dev3.
- Chrome ✅
- Firefox ✅
- Safari ✅
